.github/workflows/build-test-lint.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Build Test Lint'
+run-name: Build Test Lint of ${{ github.ref_name }} by @${{ github.actor }}
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        type: boolean
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
+
+jobs:
+  build:
+    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.16
+    secrets: inherit
+    with:
+      platforms: 'linux/amd64'
+      webTarget: web
+  cypress:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Cypress e2e
+        uses: cypress-io/github-action@v6
+        with:
+          start: yarn start
+          wait-on: 'http://localhost:3000'
+  eslint:
+    needs: build
+    uses: scientist-softserv/actions/.github/workflows/lint.yaml@v0.0.13
+    with:
+      lint_cmd: docker compose run web sh -c 'yarn && yarn lint'
```

Comment:
ref: https://github.com/scientist-softserv/viva/blob/main/.github/workflows/build-test-lint.yaml#L36-L40

I had to edit the `lint_cmd` slightly because I kept getting an error about not being able to find "bash".